Z9 changes

shardool-gore opened the pull request.

shardool-gore committed:
Z9 changes

abyshukla commented on cos.adoc:
```diff
@@ -96,6 +102,12 @@ Preferences page will not display in users mailboxes: they therefore cannot
 
+ifdef::z9[]
+[IMPORTANT]
+User can change his Preferences in {modern} under Settings. Currently, disabling Preferences does not have any impact on {modern}.
```
Gender-neutral
Users can change their Preferences in {modern} under Settings. Currently, disabling *Preferences* does not have any impact on {modern}.

mogsdad replied:
AND... one sentence per line, one line per sentence.

mogsdad commented:
OpenDrive is not available in Zimbra 9, it should be removed from the documentation. We will have just Zimbra Drive.

shardool-gore committed:
Review

shardool-gore committed:
review

abyshukla committed:
Implementing feedback from Gayle

abyshukla committed:
Implementing Feedback

abyshukla committed:
Update index.adoc

Added a line break for the level offset command.

abyshukla committed:
Updated document title

Commit:
Implemented feedback for cos.adoc + cleanup

Cleaned up `adminconsole.adoc` and `cos.adoc`:
- One line per sentence, one sentence per line.
- Grammar and spelling checked with Grammarly.

Commit:
ZBUG-1401 - Added Drive installation steps

- Copied installation steps from [Release Notes for 8.8.15 P5](https://wiki.zimbra.com/wiki/Zimbra_Releases/8.8.11/P5).
- Completed rename of Drive v2 to Zimbra Drive, using a variable this time.
- Grammarly review.

mogsdad committed:
Merge branch 'develop' into PREAPPS-4190
